action.yml

```diff
@@ -50,9 +50,6 @@ inputs:
   cilium:
     description: 'Install cilium'
     required: false
```

Comment:
This file is generated so we can’t edit it here. Check upstream in the arkade project